tests/emm/services/test_git_service.py

```diff
@@ -225,6 +231,8 @@ def test_relative_directory_should_return_full_path(self, local_mock, git_servic
         directory = Path("a/relative/path")
         assert git_service._determine_directory(directory) == Path(local_mock.cwd / directory)
 
-    def test_absolute_directory_should_return_directory(self, git_service):
+    @patch(ns("local"))
+    def test_absolute_directory_should_return_directory(self, local_mock, git_service):
         directory = Path("/a/absolute/path")
-        assert git_service._determine_directory(directory) == directory
+        absolute_directory = Path(local_mock.cwd / directory)
+        assert git_service._determine_directory(absolute_directory) == absolute_directory
```

Comment:
Doing this because absolute path in windows include "C:\", we need to ensure the path is absolute in every system.